cmd/crank/install.go

```diff
@@ -120,6 +136,30 @@ func (c *installConfigCmd) Run(k *kong.Context, logger logging.Logger) error {
 		logger.Debug("Failed to create configuration", "error", warnIfNotFound(err))
 		return errors.Wrap(warnIfNotFound(err), "cannot create configuration")
 	}
+	if err = wait.PollImmediate(waitInterval, c.Wait, func() (bool, error) {
+		logger.Debug(fmt.Sprintf(msgFmtPkgWaiting, "configuration"))
+		res, err := kube.Configurations().Get(context.Background(), pkgName, metav1.GetOptions{})
+		if err != nil {
+			logger.Debug(fmt.Sprintf(errFmtFetchPkg, "configuration"), "error", err)
+			return false, errors.Wrap(err, fmt.Sprintf(errFmtFetchPkg, "configuration"))
+		}
+		var configStatus corev1.ConditionStatus
+		for _, condition := range res.Status.Conditions {
+			if condition.Type == v1.TypeHealthy {
+				configStatus = condition.Status
+			}
+		}
+		if configStatus == corev1.ConditionTrue {
+			logger.Debug(fmt.Sprintf(msgFmtPkgReady, "configuration"))
```

Comment:
Apologies for the conflicting feedback here, but I think it may actually make sense to have a set of errors for `Configuration` and a set for `Provider`, both as `const` (i.e. `msgConfigurationReady` and `msgProviderReady`). This will keep us from having to `fmt.Sprintf` in these log messages. My apologies for going back and forth on this!